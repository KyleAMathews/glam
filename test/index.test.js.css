/* do not edit this file */
.css-hwfcu5 { color:red }
.css-1nd0lsp { color:red, font-weight:var(--css-1nd0lsp-0) }
.css-1mdi6ak { color:red; font-weight:var(--css-1mdi6ak-0) }