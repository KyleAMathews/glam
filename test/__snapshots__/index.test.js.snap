// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dudupe static sections 1`] = `
Array [
  "font-weight:bold",
  "font-weight:normal",
]
`;

exports[`extracts css into a css file 1`] = `
"/* do not edit this file */
.css-hwfcu5 { color: red }
.css-1nd0lsp { color: red,; font-weight: var(--css-1nd0lsp-0) }
.css-1mdi6ak { color: red; font-weight: var(--css-1mdi6ak-0) }"
`;

exports[`injects dynamic values into a sheet 1`] = `Array []`;

exports[`receives a class and array of var values 1`] = `
Array [
  "css-1nd0lsp",
  Array [
    "bold",
  ],
]
`;

exports[`returns 2 classes for a dynamic string 1`] = `"css-1nd0lsp vars-1wb9cdj"`;

exports[`returns a class for a string 1`] = `"css-hwfcu5"`;
